tests/utils.go

```diff
@@ -2689,22 +2689,6 @@ func WaitUntilVMIReady(vmi *v1.VirtualMachineInstance, expecterFactory VMIExpect
 	return vmi
 }
 
-func WaitUntilVMIReadyWithNamespace(namespace string, vmi *v1.VirtualMachineInstance, expecterFactory VMIExpecterFactory) *v1.VirtualMachineInstance {
```

Comment:
Why are we removing this in this commit?

Reply:
Because is not needed at all if we take the namespace from the vmi, the function is not used at all.